### PR TITLE
Allow checking duplicated vehicles

### PR DESCRIPTION
**Pull request by Tadjii**
| Q           | A                                               |
|-------------|-------------------------------------------------|
| Ticket      | https://redmine.aramisauto.com/issues/49421 |
| Change type | Build |

### CODE REVIEW

File: marketplace/filter.schema.json [hunk 1]
```diff
@@ -13,6 +13,9 @@
                     "items": {
                         "type": "string"
                     }
+                },
+                "checkVehicleDuplicated": {
+                    "type": "boolean"
```
Comment (linepogl-aa) [inline marketplace/filter.schema.json]: What is the default value if the attribute is missing? Can you put it here?

Comment (Tadjii) [reply marketplace/filter.schema.json]: Yes It's false by defaull. Fixed

Comment (linepogl-aa) [reply marketplace/filter.schema.json]: I am not sure if this is what we want. I think that by default the services should be active, with the option to opt out. At least this is what we do so far.

@gmacaire-aa your opinion?

Comment (gmacaire-aa) [reply marketplace/filter.schema.json]: same opinion

Maybe we can also choose another verb than 'check' for this filtering configuration parameter. What do you think of "removeDuplicates" ("default": true) ?

Comment (Tadjii) [reply marketplace/filter.schema.json]: Good for me 👍

**Pull request merged**